superset/assets/src/visualizations/nvd3_vis.js

```diff
@@ -363,6 +363,17 @@ export default function nvd3Vis(slice, payload) {
         throw new Error('Unrecognized visualization for nvd3' + vizType);
     }
 
+    if (isTruthy(fd.show_brush) && isTruthy(fd.send_time_range)) {
+      chart.focus.dispatch.on('brush', (event) => {
+        const extent = event.extent;
+        if (extent.some(d => d.toISOString === undefined)) {
+          return;
+        }
+        const timeRange = extent.map(d => d.toISOString().slice(0, -1)).join(' : ');
```

Comment:
Can we make sure `d` is a date here as we may support non-date xAxis in the future.

Reply:
We check that in line 369, by looking for the existence of the `toISOString` method.